docs/source/getting-started/installation.rst

```diff
@@ -92,6 +94,20 @@ Note: if you encounter *Authorization Error (Error 400: invalid_request)* with t
 
 Hint: run ``az account subscription list`` to get a list of subscription IDs under your account.
 
+**Lambda**
+
+Create the file :code:`~/.lambda_cloud/lambda_keys`, go to `API Keys <https://cloud.lambdalabs.com/api-keys>`_ to generate a key, and then add the line
```

Comment:
If this is to become the first mention of Lambda, maybe have one sentence "Lambda Cloud <with link> is a cloud provider offering low-cost GPUs." since not everyone may know it.

Reply:
Good point! I will add.

Reply:
I added a short description with a link to the Lambda Labs website